data_init.rb

```diff
@@ -0,0 +1,95 @@
+require 'json'
+require './data_values'
+
+class DataInit
+  attr_accessor :books
+
+  include Data
+
+  def initialize
+    @books = Data.books
+    @people = Data.peoples
+    @books_path = './store/books.json'
+    @persons_path = './store/people.json'
+    @rentals = Data.rentals
+    @rentals_path = './store/rentals.json'
+  end
```

Comment:
- [ ] When I run the application and try to create any data the application crashes because on these lines the app is looking for these files and they don't exist. Please make sure to handle the case when one or more of the files are missing as requested in the project requirements.
![image](https://user-images.githubusercontent.com/83803414/188899717-8f6945a0-3379-4c9b-864b-0009bc421742.png)

Reply:
Hello thanks for providing your feedback, I just want to let you know that those tests concerning files existence are being executed on the functions in the data init class there's a function for each type of data that checks whether the file exists or not `_init`, you were met with this issue because we have deleted the directory that would hold the JSON data for those files.
https://github.com/Zelalem1222/school-library/pull/8/files#diff-e19cdc73154a0df0f8b3f7cb0d50a66cb74ae460efd9fda87b44837494bf6e9bR18-R28